data/scripts/weapons/#example.lua

```diff
@@ -0,0 +1,83 @@
+--[[
+		
+	Burst Arrow example
+
+]]
+local area = createCombatArea({
+	{1, 1, 1},
+	{1, 3, 1},
+	{1, 1, 1}
+})
+
+local combat = Combat()
+combat:setParameter(COMBAT_PARAM_TYPE, COMBAT_PHYSICALDAMAGE)
+combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_EXPLOSIONAREA)
+combat:setParameter(COMBAT_PARAM_BLOCKARMOR, true)
+combat:setFormula(COMBAT_FORMULA_SKILL, 0, 0, 1, 0)
+combat:setArea(area)
+
+local burstarrow = Weapon(WEAPON_AMMO)
+
+burstarrow.onUseWeapon = function(player, variant)
+	if player:getSkull() == SKULL_BLACK then
+		return false
+	end
+
+	return combat:execute(player, variant)
+end
+
+burstarrow:id(2546)
+burstarrow:attack(27)
+burstarrow:shootType(CONST_ANI_BURSTARROW)
+burstarrow:ammoType("arrow")
+burstarrow:maxHitChance(100)
+burstarrow:register()
+
+--[[
+		
+	Wand of Vortex example
+
+]]
+local wov = Weapon(WEAPON_WAND)
+wov:id(2190)
+wov:minDamage(8)
+wov:maxDamage(18)
+wov:element("energy")
+wov:level(7)
+wov:mana(2)
+wov:vocation("sorcerer", true, true)
+wov:vocation("master sorcerer")
+wov:register()
+
+--[[
+		
+	Arbalest example
+
+]]
+local arbalest = Weapon(WEAPON_DISTANCE)
+arbalest:id(5803)
+arbalest:slotType("two-handed") -- it's now a 2h weapon
+arbalest:ammoType("bolt")
+arbalest:range(6)
+arbalest:attack(2)
+arbalest:hitChance(2)
+arbalest:level(75)
+arbalest:wieldedUnproperly(true)
+arbalest:register()
+
+--[[
+		
+	Earth Barbarian Axe example
+
+]]
+local eba = Weapon(WEAPON_AXE)
+eba:id(7859)
+eba:attack(23)
+eba:defense(18)
+eba:extraDefense(1)
+eba:extraElementAttack(5)
```

Comment:
the problem is in this case `element` and `extraElementAttack/type` are complete different cases which cannot be merged that easily without changing stuff which could break on xml parsing side, so I'd rather have them not mixed up.
I can change it so that it accepts either string like `"ice"` or `COMBAT_ICEDAMAGE`
`element` is strictly used by wands/rods, where `extraElementAttack/type` is used for weapons with enchantments and such.

Reply:
I reckoned also that would not be as easy as it seemed. Yeah, it would look better if we could also use strings.

Reply:
I'd rather stick with the enum then use strings. If allowing both isn't that much of a burden then it seems reasonable to support both.

Reply:
enabled both options in [9148138](https://github.com/otland/forgottenserver/pull/2571/commits/91481384539c767c1ab0570a52f8c011a022bb1b)